draft-terbu-sd-jwt-vc.md

```diff
@@ -300,30 +300,30 @@ Any claims used that are not understood MUST be ignored.
 Additional validation rules MAY apply, but their use is out of the scope of
 this specification.
 
-# JWT Issuer Metadata {#jwt-issuer-metadata}
+# JWT VC Issuer Metadata {#jwt-vc-issuer-metadata}
 
-This specification defines the JWT Issuer Metadata to retrieve the JWT Issuer
+This specification defines the JWT VC Issuer Metadata to retrieve the JWT Issuer
 Metadata configuration of the JWT Issuer of the JWT. The JWT Issuer is
-identified by the `iss` claim in the JWT. Use of the JWT Issuer Metadata
+identified by the `iss` claim in the JWT. Use of the JWT VC Issuer Metadata
 is OPTIONAL.
 
-JWT Issuers publishing JWT Issuer Metadata MUST make a JWT Issuer Metadata
+JWT Issuers publishing JWT VC Issuer Metadata MUST make a JWT VC Issuer Metadata
 configuration available at the path formed by concatenating the string
-`/.well-known/jwt-issuer` to the `iss` claim value in the JWT. The `iss` MUST
+`/.well-known/jwt-vc-issuer` to the `iss` claim value in the JWT. The `iss` MUST
```

Comment:
why are we intentionally breaking compatibility with oidc?

Reply:
OIDC uses a different path, so I don't think it breaks compatibility with it. (this path is being defined in this spec, because we were surprised to find that a generic jwt-issuer .well-known path has not been defined, but at the same time maybe it was not defined because it is too generic and this path needs to be scoped to jwt-vcs, not sure yet)

Reply:
@OR13 A SD-JWT VC is not an ID Token and a verifier does not know how a SD-JWT VC was issued. So I don't see the relationship to OIDC: Could you please explain what "breaking compatibility with OIDC" means?